lessons/lesson1.md

```diff
@@ -1,127 +1,38 @@
-## Lesson 1
+## Lesson 1: Create a New Firefly App from Campaign Standard Template
```

Comment:
As pointed out by Jie, should we stick to format `Lesson 1: title`, `Lesson 2: title` etc ?

Reply:
I don't have a strong opinion about numbering lessons, and short labs may not necessarily have them. This has numbering because it is quite long.

Reply:
+1 for Numbering all lessons